src/shared/debug/launchConfiguration.ts

```diff
@@ -91,7 +92,11 @@ class DefaultDebugConfigSource implements DebugConfigurationSource {
     }
 
     public async setDebugConfigurations(value: vscode.DebugConfiguration[]): Promise<void> {
-        await this.launch.update('configurations', value)
+        try {
+            await this.launch.update('configurations', value)
+        } catch (e) {
+            getLogger().error('setDebugConfigurations failed: %O', e as Error)
```

Comment:
Is this related to the PR or just for intermediate troubleshooting?

Reply:
updated to use `showMessage()` if this fails.